### ISSUE TITLE
mutation data KM plot grouped by mutation impact

### ISSUE THREAD

**Issue by jingchunzhu** [ucscXena/ucsc-xena-client#414]
https://xenabrowser.net/heatmap/?bookmark=57ff4a50af0df94c790dc89140a2fe3b run KM analysis on column D, you get the screen shot below: http://dev.xenabrowser.net/heatmap/?bookmark=85887cbe7ba6b68407350bb3c68f7e95

The KM plot curves are currently grouped into two groups, one with mutation (purple), one without mutation (white).
![image](https://user-images.githubusercontent.com/999493/55183051-392b1580-514c-11e9-8cf6-73b8e84f5fc0.png)

Desired change:
KM plot is grouped into multiple groups, the groups should be consistent with the categories in the column D spreadsheet view. Also it needs to add a category of "no mutation" which those samples that are currently in the "no mutation" category on production (i.e. the white curve above). 

Essentially, the desired change to the KM plot is to keep the "no mutation" curve, but further split the "Has mutation" curve by their impact category. 

![image](https://user-images.githubusercontent.com/999493/55183299-c1111f80-514c-11e9-8f35-93315e0e0d43.png)

**Comment by Gauravsahadev** [comment]
Can I work on this issue?

**Comment by maryjgoldman** [comment]
Yes though know that we're a bit tied up with other obligations these next few weeks so may not be as responsive as usual

**Comment by Gauravsahadev** [comment]
Okay, that won't be a problem
